src/core/tests/frontend/paddle/test_models/gen_scripts/generate_matrix_nms.py

```diff
@@ -0,0 +1,405 @@
+#
```

Comment:
Add copyright info

Reply:
Do you have the conpyright info example which is approved?

Reply:
Here is an [Example](https://github.com/openvinotoolkit/openvino/pull/10627/files#diff-4382bc5308f8880aefca6ce2743309a7dbfc048be8e49e5919da6ea8d5a5852e).